.github/scripts/setup_runners_service.sh

```diff
@@ -275,7 +275,7 @@ then
 
   if [ "$SETUP_DISTRO" = "zip" ]
   then
-    $KIBANA_ROOT/bin/kibana-plugin remove opendistro_security 
+    $KIBANA_ROOT/bin/kibana-plugin remove opendistroSecurityKibana 
     sed -i /^opendistro_security/d $KIBANA_ROOT/config/kibana.yml
```

Comment:
This has to be changed to sed -i /^opendistroSecurityKibana/d $KIBANA_ROOT/config/kibana.yml , as opendistro_security will no longer exist in the file.

Reply:
The settings have not changed. Just the plugin name